README.rst

```diff
@@ -15,6 +15,22 @@ reserved.
 
 See the end of this file for further copyright and license information.
 
+General Information
+-------------------
+
+- Website: https://www.python.org
+- Source code: https://github.com/python/cpython
+- Issue tracker: https://bugs.python.org
+- Documentation: https://docs.python.org
+- Developer's Guide: https://cpython-devguide.readthedocs.io
+
```

Comment:
The readthedocs URL is proposed here but the docs.python.org is proposed in the changes for Doc/README.rst and further down in this file.  I say stick with the latter.